### PR TITLE
Completed css-flexbox excercise.

### PR DESCRIPTION
**Pull request by phong4**
Screen Shot 2021-12-02 at 10 32 46 AM
Screen Shot 2021-12-02 at 10 32 41 AM
Screen Shot 2021-12-02 at 10 32 36 AM
Screen Shot 2021-12-02 at 10 32 31 AM

### CODE REVIEW

**Comment by phong4** [comment]
My apologies Cody as this is out of order:

the order of the screenshots is 

Display HTML
Align Items
Just Content
Flex Wrap

Please let me know if these needs to be resubmitted in correct order

File: css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        padding: 2rem;
+        flex-direction: row;
```
Comment (codingcodymiller) [inline css-flexbox/display-flex.html]: `flex-direction: row` is the default value, this can be removed.

File: css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        padding: 1rem;
+      }
+
+      .card-wrapper {
+        flex-basis: calc(100% / 6);
+        flex-direction: row;
```
Comment (codingcodymiller) [inline css-flexbox/flex-wrap.html]: The `.card-wrapper` element is not a flex container, so this property is doing nothing, it can be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by phong4** [comment]
Screen Shot 2021-12-02 at 10 52 34 AM
Screen Shot 2021-12-02 at 10 52 15 AM

**Comment by phong4** [comment]
Hi Cody, I have made the appropriate changes - Thanks!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**